zipkin-common/src/main/scala/com/twitter/zipkin/common/Span.scala

```diff
@@ -20,25 +20,35 @@ import com.twitter.zipkin.Constants
 import com.twitter.zipkin.util.Util._
 
 /**
- * A span represents one RPC request. A trace is made up of many spans.
+ * A trace is a series of spans (often RPC calls) which form a latency tree.
  *
- * A span can contain multiple annotations, some are always included such as
- * Client send -> Server received -> Server send -> Client receive.
+ * <p/>Spans are usually created by instrumentation in RPC clients or servers, but can also
+ * represent in-process activity. Annotations in spans are similar to log statements, and are
+ * sometimes created directly by application developers to indicate events of interest, such as a
+ * cache miss.
  *
- * Some are created by users, describing application specific information,
- * such as cache hits/misses.
+ * <p/>The root span is where [[traceId]] = [[id]] and [[parentId]] is empty. The root span is
```

Comment:
What specifically depends on `traceId == id` condition for the root span?  It seems very artificial, why is it not enough to have empty parent ID?